### DISCUSSION TITLE
Big mono repo

### DISCUSSION THREAD

**Discussion by ns-ggeorgiev** [renovatebot/renovate#16185] [Request Help] [answered]
Hi,

I am having a big mono-repo with so many dependencies and issues - github issues cannot handle the size of the report. The report is 4-5 times bigger than the limit of 64K. 

I disabled several major managers, to be able to fit the dashboard in the limit, but this is far from ideal.

What is the best way to handle this situation?

**Comment by rarkins** [comment]
Does it correctly truncate the content and create the issue? And if so then at which point does it truncate? Eg after branches and midway through dependencies?

**Reply by ns-ggeorgiev** [reply]
It does not truncate at all. It fails to create it reporting error. Unfortunately it seems the logs are already removed, but it was something of sort that the API failed because the body exceed the size limit of 64k.

I noticed that you have some logic to truncate PR message body. I do not see the same for the dashboard issue body though.

**Reply by rarkins** [reply]
In that case please create a bug report that issues are not being truncated, assuming you can reproduce it

**Reply by HonkingGoose** [reply]
@rarkins It looks like this issue already mentions a character limit for the Dependency Dashboard?

- https://github.com/renovatebot/renovate/issues/15808#issuecomment-1145361873

**Reply by rarkins** [reply]
If we're not truncating issue length (such as for dashboard) then we should classify it as an important bug issue to fix

**Reply by HonkingGoose** [reply]
Do you want me to bump the priority on the existing issue, or do you want me to create a new issue?

**Reply by rarkins** [reply]
Looks like we have a reproduction in #15850. I've raised its priority to high Cc @Gabriel-Ladzaretti

**Reply by Gabriel-Ladzaretti** [reply]
I've opened a [PR](https://github.com/renovatebot/renovate/pull/16208) to address this.

**Reply by ns-ggeorgiev** [reply]
thank you for taking this with a sense of urgency.

**Comment by HonkingGoose** [comment, marked as answer]
> I am having a big mono-repo with so many dependencies and issues - github issues cannot handle the size of the report.

This problem is fixed by:

- https://github.com/renovatebot/renovate/pull/16208

Renovate will now truncate the Dependency Dashboard issue body if needed.
